examples/mesh-gateways/vpc.tf

```diff
@@ -0,0 +1,40 @@
+locals {
+  // both VPCs use the same overlapping subnet addresses
+  private_subnet_cidrs = ["10.0.1.0/24", "10.0.2.0/24"]
+  public_subnet_cidrs  = ["10.0.3.0/24", "10.0.4.0/24"]
+}
+
+data "aws_availability_zones" "available" {
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
+}
+
+module "dc1_vpc" {
```

Comment:
It isn't a big deal, but we could use [`count`](https://www.terraform.io/language/meta-arguments/count#basic-syntax) to reduce some of the duplication here.

Reply:
I will include this in my next PR for enabling ACL support.